client/landing/jetpack-cloud/components/sidebar/index.jsx

```diff
@@ -48,9 +48,7 @@ class JetpackCloudSidebar extends Component {
 		const { selectedSiteSlug, translate } = this.props;
 
 		return (
-			<Sidebar
-				className="is-jetpack-cloud-sidebar" // eslint-disable-line wpcalypso/jsx-classname-namespace
-			>
+			<Sidebar className="sidebar sidebar__jetpack-cloud">
```

Comment:
The `sidebar` class will be [added anyway](https://github.com/Automattic/wp-calypso/blob/d77f51b8a4cd5468e062028a8e0a8a70e3f2fe04/client/layout/sidebar/index.jsx#L15) so it could be removed from here.

Reply:
:+1: